test-case/check-userspace-packages.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+##
+## Case Name: check-userspace-packages
+## Preconditions:
+##    N/A
+## Description:
+##    check the userspace packages building and install the packages
+## Case step:
+##    1. download latest alsa-lib, pulseaudio debian packages and UCM
+##    2. install the userspace packages and UCM
+##    3. restart pulseaudio
+## Expect result:
+##    All packages and UCM are installed successfully and pulseaudio restarts successfuly
+##
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+OPT_OPT_lst['a']='alsalib'     OPT_DESC_lst['a']='alsalib pakcages URL location'
+OPT_PARM_lst['a']=1         OPT_VALUE_lst['a']=''
+
+OPT_OPT_lst['p']='pulseaudio'     OPT_DESC_lst['p']='pulseaudio pakcages URL location'
+OPT_PARM_lst['p']=1         OPT_VALUE_lst['p']=''
+
+OPT_OPT_lst['u']='ucm'     OPT_DESC_lst['u']='UCM git repo URL location'
+OPT_PARM_lst['u']=1         OPT_VALUE_lst['u']=''
+
+func_opt_parse_option "$@"
+
+dat=`date +%Y%m%d`
+ART_URL=http://sof-ci-storage.sh.intel.com
```

Comment:
This needs to be configurable to be usable outside Intel.

Reply:
I do not think this function should not be a test case. This should be some ENV check for our CI instead of a shared test case. Please use dpkg or apt for check only. Install should be in other part.